osgar/record.py

```diff
@@ -89,7 +89,8 @@ def main(record=record):
     parser.add_argument('--log', help='force record log filename')
     parser.add_argument('--application', help='import string to application', default=None)
     parser.add_argument('--params', nargs='+',
-                        help='optional list of configuration parameters like app.max_speed=0.1 app.dist=-2.0')
+                        help='optional list of configuration parameters like app.max_speed=0.1 app.dist=-2.0. '
+                             'For string parameters use double quotes e.g.: app.logfile=\'"name.log"\'')
```

Comment:
OK, thanks - I did not realized that it is not working directly. Maybe we may try type checking from the original JSON?? Let's close it here, but we may came to it again soon.